### PR TITLE
quickstart corrections

### PR DESCRIPTION
**Pull request by AnnuCode**
Added some corrections in the Quickstart guide.

### CODE REVIEW

File: cli/getting-started/index.md [hunk 1]
```diff
@@ -69,26 +69,24 @@ https://github.com/terramate-io/terramate/releases
...
+Other installation options [are available](../installation.md#alternative-installation-methods). When the installation is complete, you can test it out by reading the current version:
 
 ```sh
 $ terramate version
 ```
 
 ### Install Terraform or OpenTofu
 
-Next, install Terraform or OpenTofu which we will orchestrate using Terramate:
+Next, install Terraform or OpenTofu to orchestrate it using Terramate:
```
Comment (soerenmartius) [inline cli/getting-started/index.md:80]:
```suggestion
Next, install Terraform or OpenTofu:
```

sounds weird, maybe we just ditch orchestration here?

File: cli/getting-started/index.md [hunk 2]
```diff
@@ -69,26 +69,24 @@ https://github.com/terramate-io/terramate/releases
...
 
 ## Create a new project
 
-Terramate requires a git repository to work and every git repository is considered
-a project in Terramate.
+Terramate requires a git repository, and every git repository is a project in Terramate.
 
-Let's set up your first Terramate project by creating a new directory `terramate-quickstart` and initializing a new, empty
-repository:
+Let's create a new directory called `terramate-quickstart` and initialize Git to set up your first Terramate project.
```
Comment (soerenmartius) [inline cli/getting-started/index.md:89]:
```suggestion
Let's create a new repository `terramate-quickstart` to set up your first Terramate project.
```

File: cli/getting-started/index.md [hunk 3]
```diff
@@ -576,7 +572,7 @@ We hope this tutorial has helped you grasp the basics of Terramate. Here's a sum
 - How to create a [Terramate Cloud](https://cloud.terramate.io) account and connect it to your project.
 
 Those examples hopefully give you a starting point and help you get insights into the capabilities of Terramate.
-This is just the tip of the iceberg and running code generation and change detection on scale can help you save a lot of time when maintaining or running Terraform.
+This quickstart is just the tip of the iceberg. Running code generation and change detection on the scale can help you save a lot of time when maintaining or running Terraform.
```
Comment (soerenmartius) [inline cli/getting-started/index.md:575]:
```suggestion
This quickstart is just the tip of the iceberg. Running code generation and change detection can help you save a lot of time when maintaining or running Terraform.
```

**Review by soerenmartius — CHANGES_REQUESTED** [review]

File: cli/getting-started/index.md (hunk 1 above)
Comment (AnnuCode) [reply cli/getting-started/index.md:80]: yes, I think it's not needed here.

**Review by soerenmartius — APPROVED** [review]

**Pull request merged**